src/Plugin/Model/PdkOrder.php

```diff
@@ -90,22 +90,26 @@ public function __construct(?array $data = [])
      * @return \MyParcelNL\Pdk\Shipment\Model\Shipment
      * @throws \MyParcelNL\Pdk\Base\Exception\InvalidCastException
      */
-    public function createShipment(array $data = []): Shipment
+    public function createShipments(array $data = []): Shipment
```

Comment:
Dat createShipments (meervoud) 1 shipment teruggeeft zit mij niet lekker, het is niet transparant bij het aanroepen wat hij dan create en welke hij dan teruggeeft?